includes/reader-revenue/woocommerce/class-woocommerce-connection.php

```diff
@@ -14,9 +14,10 @@
  */
 class WooCommerce_Connection {
 	/**
-	 * Statuses considered active subscriptions.
+	 * Statuses considered active.
 	 */
 	const ACTIVE_SUBSCRIPTION_STATUSES = [ 'active', 'pending', 'pending-cancel' ];
+	const ACTIVE_ORDER_STATUSES = [ 'processing', 'completed' ];
```

Comment:
I know it's been like this before, but why are we including the `processing` status here? At this stage, the order could fail and the `order_completed` data event only sends orders with the status `completed`.

Reply:
Nevermind! Just read on Woo docs:
> Processing: Payment has been received (paid), and the stock has been reduced. The order is awaiting fulfillment.